app/templates/components/section.html

```diff
@@ -10,31 +10,16 @@ <h2 class="text-sm font-bold mt-0 mb-2 text-corporate-blue pb-10">
       {{ section.title }}
     </h2>
     <h3 class="text-corporate-blue text-2xl font-thin leading-10 pb-10">{{ section.disclaimer }}</h3>
-    <p class="font-source">{{ section.sectionDescription | safe }}</p>
-    <div class="my-4">
-      {% set chart_ids = section.get("chart-ids", []) %}
-      {% if chart_ids|length > 1 %}
-        {% with chart_ids=chart_ids %}
-          {% with section_id=section_id %}
-            {% with section_title=section.title %}
-              {% with toggleText=section.toggleText %}
-                {% include "components/chart-toggle.html" %}
-              {% endwith %}
+    <div class="mb-4">
+      {% set charts = section.charts %}
+      {% if charts %}
+        {% with charts=charts %}
+          {% for chart in charts %}
+            {% with chart = chart %}
+              {% include "components/chart-toggle.html" %}
             {% endwith %}
-          {% endwith %}
+          {% endfor %}
         {% endwith %}
```

Comment:
I think this can be simplified even further now (unless we need more complexity here again int the future):

```
      {% set charts = section.charts %}
      {% for chart in charts %}
        {% include "components/chart-toggle.html" %}
      {% endfor %}
```

Reply:
doner, merci!